src/types.js

```diff
@@ -53,3 +47,41 @@ export type Persona = {
   unknownCommand: string,
   error: string
 };
+
+export type LandRequest = {
+  pullRequestId: string,
+  username: string,
+  userUuid: string,
+  pullRequestState?: 'OPEN' | 'DECLINED' | 'MERGED',
```

Comment:
I'm not actually using this key yet, but we'll probably like to add it once there's more history/ui around this.